### PR TITLE
First draft of "publications" extension

### PR DESCRIPTION
**Pull request by apdavison**
The primary aim of this extension is to support Live Papers, however the schemas could be used anywhere a more fine-grained representation of a publication is needed (rather than just a DOI).

### CODE REVIEW

File: schemas/scholarlyArticle.schema.tpl.json
Comment (UlrikeS91) [inline schemas/scholarlyArticle.schema.tpl.json]: If the `Volume` is an embedded type for the `Issue`, you should remove volume from.

File: schemas/livePaperSectionType.schema.tpl.json
Comment (UlrikeS91) [inline schemas/livePaperSectionType.schema.tpl.json]: Since this is supposed to be a controlledTerm, the schema would need to be deposited in the controlledTerm repo (+ added to the terminologies + an instance folder should be started as well). So, the `type` would be "https://openminds.ebrains.eu/**controlledTerms**/LivePaperSectionType" instead. The links in other schemas would need to be adjusted then, of course :)

File: schemas/creativeWork.schema.tpl.json
Comment (UlrikeS91) [inline schemas/creativeWork.schema.tpl.json]: I misunderstood the property name (`ciatation`) at first. The instructions clarified it, though. Maybe you could find another property name? 
e.g. "referencedPublication" (along the lines of "relatedPublication" maybe confusing) or just simple "references" (or would this also be a little ambiguous in itself?)

File: schemas/creativeWork.schema.tpl.json
Comment (UlrikeS91) [inline schemas/creativeWork.schema.tpl.json]: I wonder if this is necessary. The research products have "relatedPublications". At the moment, they point to a DOI or ISBN, but I don't see why this list cannot be expanded to included creativeWorks as well. That would create the same link but would also allow that other research products can use the publication extensions, e.g. the openMINDS research product linking to the openMINDS scholarlyArticle ;)

File: schemas/chapter.schema.tpl.json
Comment (UlrikeS91) [inline schemas/chapter.schema.tpl.json:14]: I'm  curious if the schema name could maybe be changed to "bookChapter" since can only be part of a "book", or is this just at the moment like this? Are you planning to allow more than just book under "isPartOf"?

File: schemas/scholarlyArticle.schema.tpl.json
Comment (apdavison) [reply schemas/scholarlyArticle.schema.tpl.json]: Not all volumes are divided into issues, so we could have "article->issue->volume" or just "article->volume"

File: schemas/livePaperSectionType.schema.tpl.json
Comment (apdavison) [reply schemas/livePaperSectionType.schema.tpl.json]: indeed. I decided to put them here for now so that everything is in one place for reviewing, but I agree they should be moved.

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json]: The property name is taken from schema.org ([citation](https://schema.org/citation)) - I tried to stay as close to schema.org as possible to promote interoperability and avoid reinventing the wheel.

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json]: I think you're right, it's not necessary, it's one of the properties from schema.org that I thought might be useful to keep.

Adding CreativeWork to the possible types for "relatedPublications" would require CreativeWork to be in core, otherwise there would be a dependency of core on an extension. However, since CreativeWorks generally have a DOI or ISBN we can just use those, so I will remove this property here.

File: schemas/chapter.schema.tpl.json
Comment (apdavison) [reply schemas/chapter.schema.tpl.json:14]: this is again a case where I just used the name from schema.org. I think it's clear from the context that this refers to a book chapter and not a religious community 😉

**Comment by apdavison** [comment]
many thanks for the comments @UlrikeS91 ! I'll wait for any comments from @lzehl and @olinux and then make all the changes in a single commit.

File: schemas/livePaperSectionType.schema.tpl.json
Comment (UlrikeS91) [reply schemas/livePaperSectionType.schema.tpl.json]: Yeah, that makes sense :)

File: schemas/creativeWork.schema.tpl.json
Comment (UlrikeS91) [reply schemas/creativeWork.schema.tpl.json]: I'm not sure if this dependency to an extension would be bad, and we kind of do the same thing with SANDS. SANDS could also be viewed as an extension in a way. @lzehl what do you think?

File: LICENSE
Comment (lzehl) [inline LICENSE]: we agreed now that we list here all currently involved institutes? or should we split for now as temporary place holder until we set up a proper openMINDS initiative?

**Comment by apdavison** [comment]
@lzehl I'd like to migrate the live papers from KG v2 to KG v3, so please could you take a look at this soon?

File: schemas/book.schema.tpl.json
Comment (lzehl) [inline schemas/book.schema.tpl.json]: should be adjusted since a "book" in the context here is not registered as a research product. 

"Add the globally unique and persistent digital identifier of this book."

File: schemas/book.schema.tpl.json
Comment (lzehl) [inline schemas/book.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [reply schemas/creativeWork.schema.tpl.json]: I agree with removing this property here and potentially adding creative work schemas as options to relatedPublications.

Theoretically though all creative works should be indirectly linked to the relatedPublications of the research products via their digital identifiers (DOI, ISBN, ISSN, etc). This would also avoid the issue if we should allow dependencies of openMINDS_core to extensions (which I think are probably not fully avoidable).

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json:7]: should this not be "corresponding author" for creative works?

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [reply schemas/creativeWork.schema.tpl.json]: I rather have an issue here of what should be entered. 

E.g. would this require for a Book to enter all references that are made in the book to another source? If yes I would probably delete this property completely and leave it to the actual creative work to define this. At least this should be optional and make clear that it should either be complete or left empty.

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: is this really a point of interest? I would keep only the release date (but we can keep it of course)

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: this should be specific for creative work and not mix up with research products. OR this whole extension needs to be revised under the consideration of either "research products" == "creative works" or "research products" != "creative works". (see also general comment: I find having this in parallel a bit confusing)

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ??? (and here also inconsistent without "_")

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json]: ???

**Review by lzehl — COMMENTED** [review]
@apdavison this is a great extension. I have some questions / comments that we need to solve for an openMINDS integration.

The purpose of "_originalName" is unclear to me. If this is a reference to schema.org, please define this part in properties.json in the main openMINDS GitHub.

Generally speaking I find the definition of Creative Work in parallel / addition to the Research Products a bit confusing. If we do this there should be a very clear difference in the metadata between them. You also are only using this for defining the specifications expected behind a digital identifier of something that is published. While the LivePaper(Version) are actually an extension for ResearchProduct(Version).

**My thoughts / suggestion to revise this a bit and :**

I think this extension should hold LivePaper(Version) as new ResearchProduct(Version) as you do it right now. Feel free to add missing elements that in principle are also true (but barely used) for the other research products from to the concept schema in core as optional properties (if there are any).

Note that linkages of openMINDS schemas or properties to schema.org schemas or properties should be made through the type.json or property.json files in the main openMINDS GitHub, the central solution for referencing any other metadata initiative/standard.

Revise Book, Chapter, etc. as schemas that extend the digital identifiers (DOI, ISSN, ISBN, etc.). Creative Work could be the concept schema here linking to the digitial identifier for which they provide more structured information (e.g. specifying that a DOI is actually a Book, Chapter, or PeerReviewedPaper with respective metadata information).
To clarify:
Related publications in the research product (versions) links to digitial identifiers. Each digital identifier can be also linked from an creative work schema to add additional structured information for a digital identifier. 
Sketch: Dataset (via relatedPublication) - > DOI <- Book (via ditialIdentifer)

Does this make sense? Do you like the idea? It is only a slightly different approach to what you implemented in the first draft.

File: LICENSE
Comment (apdavison) [reply LICENSE]: I thought it was all currently involved institutes on a per-module level (rather than all-of-openMINDS), i.e. as soon as we have a git commit from someone from U. Oslo in the v1 branch of this repository, we would add U. Oslo to the copyright statements...

I think this sends the right message as far as encouraging contributions from outside HBP.

File: schemas/book.schema.tpl.json
Comment (apdavison) [reply schemas/book.schema.tpl.json]: fixed in next push

File: schemas/book.schema.tpl.json
Comment (apdavison) [reply schemas/book.schema.tpl.json]: sorry, the "_originalName" field should have been deleted. Will be gone in the next push.

These schemas are closely based on schema.org, and where I changed a field name to use an equivalent openMINDS term I used this field to keep a note for myself of the original schema.org name, but the field wasn't intended to go into the v1 branch.

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json:7]: I was trying to reuse existing openMINDS terms, I think "corresponding author" has effectively the same role for a journal article as "custodian" for a dataset.

I'm happy to change it, if you think "custodian" is confusing.

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json]: about the "citation" field, it is optional.

about the "creationDate" field, we could remove it. It's often helpful to have a date field associated with unreleased items, however, which is why I left it in (from schema.org)

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json]: yes, sorry, this was a lazy copy-and-paste without enough editing afterwards. Fixed in next push.

**Comment by apdavison** [comment]
@lzehl and @UlrikeS91 I've made changes to implement most of your individual suggestions.

I guess I need to make a PR to update properties.json and types.json in the parent openMINDS repository, or are such updates automated?

I also need to make a PR to move LivePaperSectionType to controlledTerms, yes?

On the issue of related publications, I'm not sure I fully understand your proposal, Lyuba, but in the context of this PR, it seems the only remaining change that would be needed is to change the "_linkedTypes" for "LivePaper/relatedPublication" to ["https://openminds.ebrains.eu/core/DOI", "https://openminds.ebrains.eu/core/ISBN", "https://openminds.ebrains.eu/core/ISSN"]. Is that correct?

**Comment by lzehl** [comment]
@apdavison I'll have a look at your changes later or tomorrow. For your questions:

The properties.json and types.json are automatically updated with the property name and where it is used in openMINDS. The relations to other metadata standards have to be manually added and committed (we did not really do this so far; we need focus on this a bit more in future). Let me know if you do not find proper documenation on how to do this. 

 LivePaperSectionType should indeed be added to controlledTerms via a PR yes.

And yes, my proposal would indeed only require the "_linkedTypes" for "LivePaper/relatedPublication" to be ["https://openminds.ebrains.eu/core/DOI", "https://openminds.ebrains.eu/core/ISBN", "https://openminds.ebrains.eu/core/ISSN"]. Potentially we can adapt the metadata for the creative work schemas a bit. They do not have to follow any concept of the research product, but can fully orient themselves to the most common way of defining individual creative work information (e.g. that are typically also registered by the digital identifier service side, e.g. DataCite). I'll have a closer look at this tomorrow.

**Comment by apdavison** [comment]
@lzehl that would be great, thanks.

I'm happy to adapt the creative work schemas. To be clear, they weren't really influenced by the research product schema, but were based on schema.org, with some property names adapted where equivalent terms already existed in openMINDS.

In comparing schema.org and DataCite, these pages seem like they might be useful:
- https://support.datacite.org/docs/how-can-i-map-different-metadata-formats-to-the-datacite-xml
- https://support.datacite.org/docs/schemaorg

File: LICENSE
Comment (lzehl) [reply LICENSE]: As said: openMINDS has a central repository for access and maintenance of vocabulary. Last time we discussed this we both agreed that all repositories should have the same copyright holders to not make the licenses more complicated than necessary. Until we introduce an openMINDS initiative as copyright holder I think we should go with all current institutes then (and actually also correct this for the other repos), no? Or each repo for now maintains their own copyright, the main one needs to be updated with a merged copyright and we replace everything with openMINDS once we set up the initiative.

**Review by lzehl — CHANGES_REQUESTED** [review]
Suggestions for CreativeWork concept schema based on the metadata of DataCite and schema.org:

**Required properties:**
+ **creator** (DataCite: creator; schema.org: author)
+ **datePublished** (DataCite: publicationYear; schema.org: datePublished)
+ **digitalIdentifier** (DataCite: identifier; schema.org: at_id)
+ **license** (DataCite: rights; schema.org: license)
+ **publisher** (DataCite: publisher; schema.org: publisher)
+ **title** (DataCite: title; schema.org: name)

**Optional properties:**
+ **description** (DataCite: description; schema.org: description)
+ **dateCreated**
+ **dateModified**
+ **editor** (might also go in the extensions)
+ **funding** (might also go in the extensions)
+ **keyword**
+ **URL**
+ **versionIdentifier** (DataCite: version; schema.org: version) (might also go in the extensions)
+ **relatedPublication** (DataCite: relatedIdentifier; schema.org: citation)

Each resourceTypeGeneral (DataCite) would extend the CreativeWork concept schema and add some more specific metadata (e.g., isPartOf, or adapt selection of digitalIdentifier). DataCite lists the following resourceTypeGeneral (I've marked the one we have and the one we need):
+ Audiovisual
+ **Book** (needed)
+ **BookChapter** (needed)
+ Collection
+ **ComputationalNotebook** (needed)
+ **ConferencePaper** (needed)
+ **ConferenceProceedings** (needed)
+ **DataPaper** (needed)
+ **Dataset** (exists as Dataset research product)
+ **Dissertation** (needed)
+ Event
+ Image
+ **InteractiveResource** (will exist as LivePaper research product)
+ **Journal** (needed)
+ **JournalArticle** (needed)
+ **Model** (exists as Model research product)
+ OutputManagementPlan
+ **PeerReview** (needed)
+ PhysicalObject
+ **Preprint** (needed)
+ **Report** (needed)
+ **Service** (needed)
+ **Software** (exists as Software research product)
+ Sound
+ **Standard** (exists as MetaDataModel research product)
+ Text
+ **Workflow** (exists as Workflow research product)
+ Other

Of course we don't have to have them all in the first version of the publication extension. I would stick to the name of the types of DataCite (or the corresponding schema.org types). The one you currently integrated are Book, BookChapter (you called it Chapter), JournalArticle ? (you called it ScholarlyArticle). Feel free to add here what you think is most need in the first round.

Independent of this PR we should group all digitalIdentifiers in core into a folder called digitalIdentifier. We will add more in future (see Appendix table 8 in https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf) and it would be good to organize them. This should be an easy change because it should not affect the pipeline or any other tooling (hope this is correct for fairgraph as well ?).

Besides the copyrights (minor issue, let's just go with one agreement and adapt all other repos) and the edition of creative work + extending schemas I think this looks good.

**Comment by apdavison** [comment]
All good with me, except:

#### Property names:
- **creator** - we already use author in openMINDS/core, so I suggest sticking with that
- **relatedPublication** - this is broader than just citations, so I would like to stick with "citation"
- **title** - for research products we use "fullName". Wouldn't it be better to use that for consistency?

#### Required fields:
- **publisher** - for journals we don't really care in general who the publisher is, so I would leave this optional to avoid unecessary work. We could make it required for Book.
- **license** - non open-access articles/books don't in general have a licence
- **digitalIdentifier** - unless I'm mistake, older articles don't always have a DOI

#### Resource types:
The ones I've included in this PR are the ones needed for the live papers migration, so I'd prefer to merge this PR once we've agreed on the properties as above, and then add others as needed in future PRs.

I used [ScholarlyArticle](https://schema.org/ScholarlyArticle), rather than JournalArticle, to match schema.org. I think the schema covers journal articles, preprints, maybe conference papers. Do you think we should create separate schemas for these?

**Comment by apdavison** [comment]
One more point to discuss: we could simplify the schemas by removing PublicationVolume and PublicationIssue, and instead adding fields "publicationVolume" and "publicationIssue" to ScholarlyArticle. What do you think?

**Comment by lzehl** [comment]
@apdavison I have some questions / comments then:

**creator**: we actually use "author" as well as "developer" in openMINDS depending on the research product. Since Creative Work is a quite broad term I thought it is also smart sticking with the more broader term "creator". But I leave this to you.

**citation**: I have no idea what you expect the value to be here then. Could you explain?

**title**: we use fullName for the research products because we also expect there to be a shortName. The Creative Work types typically do not have an alternative short name. If you prefer you can use "name" which we use in other openMINDS schemas where only one name is to be expected.

For the required fields: do as you suggest, I agree. For older articles: they do have other identifiers though, correct? So maybe the digitalIdentifier just needs to incorporate those? (might require to change the property name into "persistentIdentifier" if they are not digital)

**Comment by apdavison** [comment]
"citation" - like "relatedPublication" but only for a specific type of relationship, i.e. creative works that are cited by the current creative work

We could use "citedPublication" for clarity, the schema.org term is easy to confuse with "howToCite"

*or*, we could introduce a way to specify the type of the relationship, but I think that would probably be overkill for the time being

File: schemas/book.schema.tpl.json
Comment (lzehl) [inline schemas/book.schema.tpl.json:5]: should the "digitalIdentifier" not also be a required field?

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json:96]: might be more than that. we might need to introduce a "keyword" category (not needed for this PR though)

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [inline schemas/creativeWork.schema.tpl.json:108]: should a journal be also an option here or is this registered as organization then (if yes why do we need periodical schema in addition) ?

File: schemas/livePaperResourceType.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperResourceType.schema.tpl.json]: needs to move to controlledTerms completely

File: schemas/livePaperSectionType.schema.tpl.json
Comment (lzehl) [reply schemas/livePaperSectionType.schema.tpl.json]: don't forget to move this completely to controlledTerms

File: schemas/livePaper.schema.tpl.json
Comment (lzehl) [inline schemas/livePaper.schema.tpl.json]: this is currently only reserved for ResearchProducVersions. If we want this for the ResearchProducts we can add this centrally in core.

File: schemas/livePaperVersion.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperVersion.schema.tpl.json]: a ResearchProductVersion already has an abstract field called "description"

File: schemas/livePaperVersion.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperVersion.schema.tpl.json]: what is to be captured here? for all other research products we decided in the past that only the date where the item is publicly released is relevant. If we need to have creation date captured we should do this for all ResearchProductVersions

File: schemas/livePaperSection.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperSection.schema.tpl.json]: formally each item of the list should be in its own line

File: schemas/scholarlyArticle.schema.tpl.json
Comment (lzehl) [reply schemas/scholarlyArticle.schema.tpl.json]: but embedding them does not make sense to me because multiple articles are part of the same issue or same volume. so they need to be independently registered as linked types so that multiple article can use them

File: schemas/publicationIssue.schema.tpl.json
Comment (lzehl) [inline schemas/publicationIssue.schema.tpl.json:15]: should be linked type since multiple issues can be part of the same volume

File: schemas/livePaperResourceItem.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperResourceItem.schema.tpl.json]: formally items should be listed in seperate lines

**Review by lzehl — CHANGES_REQUESTED** [review]
@apdavison only small things now I think and some decisions we need to make considering changes in core. 

One question / comment: PublicationIssue, and PublicationVolume could potentially hold more information, correct? We don't have to add them for this PR but I was wondering what other metadata could be of interest here.

File: schemas/book.schema.tpl.json
Comment (apdavison) [reply schemas/book.schema.tpl.json:5]: looking up ISBNs of books when all you want to do is cite one chapter from the book is a tedious business, so I'd prefer to make it optional.

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json:96]: ok

File: schemas/creativeWork.schema.tpl.json
Comment (apdavison) [reply schemas/creativeWork.schema.tpl.json:108]: a journal is not a publisher, e.g. the journal Nature is published by the organization Springer-Nature

File: schemas/livePaperVersion.schema.tpl.json
Comment (apdavison) [reply schemas/livePaperVersion.schema.tpl.json]: ok, I've removed this

File: schemas/livePaper.schema.tpl.json
Comment (apdavison) [reply schemas/livePaper.schema.tpl.json]: I've moved it to LivePaperVersion

File: schemas/livePaperVersion.schema.tpl.json
Comment (apdavison) [reply schemas/livePaperVersion.schema.tpl.json]: for research products that are developed over time through an authoring UI before being released, like live papers, we really need a date field that can be used pre-publication, for sorting.

Thinking about this, though, "lastModifiedDate" would be more useful. I will make that change here, and then if you think this makes sense for other ResearchProducts we can move it to core in a future PR.

File: schemas/livePaperSection.schema.tpl.json
Comment (apdavison) [reply schemas/livePaperSection.schema.tpl.json]: done

File: schemas/publicationIssue.schema.tpl.json
Comment (apdavison) [reply schemas/publicationIssue.schema.tpl.json:15]: Having them linked would make sense if we cared whether two articles belonged to the same issue, but in practice we don't care. Having them embedded makes code simpler and queries potentially faster, since we don't have to follow the links in the graph.

An alternative, as I mentioned above, would be to remove these schemas, and just have string fields "volume" and "issue" in Periodical.

File: schemas/creativeWork.schema.tpl.json
Comment (lzehl) [reply schemas/creativeWork.schema.tpl.json:108]: okay. I was wondering about that. Thanks for clarifying

File: schemas/book.schema.tpl.json
Comment (lzehl) [reply schemas/book.schema.tpl.json:5]: okay

File: schemas/livePaperVersion.schema.tpl.json
Comment (lzehl) [reply schemas/livePaperVersion.schema.tpl.json]: sounds good. lets do it like that

File: schemas/livePaperResourceItem.schema.tpl.json
Comment (apdavison) [reply schemas/livePaperResourceItem.schema.tpl.json]: done

**Comment by apdavison** [comment]
Hi @lzehl I've pushed some changes. The only remaining issues that might need discussion are the "lastModified" field and the way we handle journal volumes and issues.

I will make a PR to controlledTerms in a few minutes

File: schemas/publicationIssue.schema.tpl.json
Comment (lzehl) [reply schemas/publicationIssue.schema.tpl.json:15]: the latter alternative I do not like because we would need to register journal information over and over again. If you want to register them directly and get rid of the schemas you should integrate them into the ScholarlyArticle schema as properties. that would be okay with me as alternative.

IMO we should stick to graph structures wherever we can and only consider code simplification where no joint information between instances is to be expected. Meaning I would make them linked types and live with the resulting query structure.

**Comment by lzehl** [comment]
@apdavison feel free to add "lastModified" for now to your schemas here. Keep note that this is automatically registered by the KG anyway in the background. For ResearchProducts and ResearchProductVersions we can make it an optional property of openMINDS though (in the long run). 

As for the journal volumes and issues: if directly integrated as properties instead of schemas please make them part of the ScholarlyArticle Schema not the Periodical Schema (because the latter is definitely reusable across many entries). I would stick with the linked_types and the complete graph structure, but for simplification I think integrating them in ScholarlyArticle is okay.

**Comment by apdavison** [comment]
re volumes and issues, sorry, I meant ScholarlyArticle, not Periodical.

For "lastModified", I'll look into extracting the automatically-added information with fairgraph, then we could remove it from the schema in a future PR.

**Comment by apdavison** [comment]
I've decided to give the linked types approach a try. If it proves too cumbersome we could always revise this in a future version.

File: schemas/livePaperVersion.schema.tpl.json
Comment (lzehl) [inline schemas/livePaperVersion.schema.tpl.json:54]: as a note: this should not be necessary to list here since it is already a property of the concept schema RPV. missing identifier types can be added there. please reconsider this in a future PR

**Review by lzehl — APPROVED** [review]
@apdavison I'm going to merge this now. Note that "relatedPublication" in LivePaperVersion could be completely removed in future and handled by the core concept schema ResearchProductVersion (adding more identifier options there was to be expected)

**Pull request merged**
